docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,35 +2,31 @@
 
 ✄ -----------------------------------------------------------------------------
 
-Thank you for your Pull Request! 🙏
-
-Before you submit, please check that:
-
-- [ ] **Description:** You added a brief description of the PR, e.g.:
-  - What does it do?
-  - What important points should reviewers know?
-  - Is there something left for follow-up PRs?
-- [ ] **Labels:** You labeled the PR appropriately if you have permissions to do so:
-  - [ ] `A*` for PR status (**one required**)
-  - [ ] `B*` for changelog (**one required**)
-  - [ ] `C*` for release notes (**exactly one required**)
-  - [ ] `D*` for various implications/requirements
-  - [ ] Github project assignment
-- [ ] **Related Issues:** You mentioned a related issue if this PR is related to it, e.g. `Fixes #228` or `Related #1337`.
-- [ ] **2 Reviewers:** You asked at least two reviewers to review. If you aren't sure, start with GH suggestions.
-- [ ] **Style Guide:** Your PR adheres to [the style guide](https://github.com/paritytech/substrate/blob/master/docs/STYLE_GUIDE.md)
-  - In particular, mind the maximal line length of 100 (120 in exceptional circumstances).
-  - There is no commented code checked in unless necessary.
-  - Any panickers in the runtime have a proof or were removed.
-- [ ] **Runtime Version:** You bumped the runtime version if there are breaking changes in the **runtime**.
-- [ ] **Docs:** You updated any rustdocs which may need to change.
-- [ ] **Polkadot Companion:** Has the PR altered the external API or interfaces used by Polkadot?
-  - [ ] If so, do you have the corresponding Polkadot PR ready?
-  - [ ] Optionally: Do you have a corresponding Cumulus PR?
-
-Refer to [the contributing guide](https://github.com/paritytech/substrate/blob/master/docs/CONTRIBUTING.adoc) for details.
-
-After you've read this notice feel free to remove it.
-Thank you!
+Thank you for your Pull Request! 🙏 Please make sure it follows the CONTRIBUTING guidelines outlined in [this document](./CONTRIBUTING.adoc) and fill out the sections below. Once you're ready to submit your PR for review, please delete this section and leave only the text under the "Description" heading.
+
+# Description
+
+*Please include a summary of the changes and the related issue. Please also include relevant motivation and context, including:*
+
+- What does this PR do?
+- Why are these changes needed?
+- How were these changes implemented and what do they affect?
+
+Fixes # (issue number, *if applicable*) 
+
+Polkadot companion: (*if applicable*)
+
+Cumulus companion: (*if applicable*)
+
+# Checklist
+
+- [ ] My PR includes a detailed description as outlined in the "Description" section above
+- [ ] My PR follows the [labeling requirements](./CONTRIBUTING.adoc#merge-process) of this project (at minimum one label for each `A`, `B`, `C` and `D` required)
+- [ ] My code follows the [style guidelines](./STYLE_GUIDE.md) of this project
```

Comment:
Why repeat stuff about the description again?

Labeling is enforced by CI and external people can not change lables?

Style guide aka fmt is also checked via CI

Reply:
How about just adding this note under the #Checklist heading:

> The CI bot will check that you have fulfilled the labeling and style guide requirements detailed in the checklist below. Don't worry if you don't have permissions to set the labels, you can still submit your PR and the person reviewing will set them for you.

Reply:
Why put there redundant information? I mean people will see this ^^

Reply:
Yeah I would remove the old style-guide. It pertains to the days that we didn't use `rustfmt` 🙈